### ISSUE TITLE
JAWS 17 not reading dropkick dropdown

### ISSUE THREAD

**Issue by RenugaP** [Robdel12/DropKick#340]
### Version of Dropkick:
jquery.dropkick-1.0.3

### Expected Behavior
JAWS should be able to read dropkick dropdowns

### Actual Behavior

JAWS screen reader not able to read dropdowns created using following dropkick template.

 dropdownTemplate = [
        '<div  class="dk_container  {{ isDisabled }}" id="dk_container_{{ id }}" tabindex="{{ tabindex }}">',
          '<a class="dk_toggle">',
            '<span class="dk_label combinationText visibleElements">{{ label }}</span>',
          '</a>',
          	'<div class="dk_options">',
            '<ul class="dk_options_inner">',
            '</ul>',
          '</div>',
        '</div>'
      ].join(''),

      // HTML template for dropdown options
      optionTemplate = '<li class="{{ current }}"><a data-dk-dropdown-value="{{ value }}">{{ text }}</a></li>',

I have tried adding role="combobox" after that it reads as combobox and not reading the options while pressing up and down arrow
